examples/default/main.tf

```diff
@@ -13,7 +13,7 @@ terraform {
 }
 
 provider "aws" {
-  version             = "1.56.0"
+  version             = "2.20.0"
```

Comment:
Same here, can just set it to `>= 2.20`. This is what I started doing, because I think we want to always test with the latest versions since its unlikely that end users will pin different versions of providers for different modules.

Reply:
Agreed 👍